admin/js/wi-volunteer-management-block.js

```diff
@@ -124,10 +127,24 @@
 		},
 	} );
 
-	// Hide the volunteer opportunities block when editing a volunteer opportunity post
-	wp.domReady( function() {
```

Comment:
This was causing a similar issue as we saw in the Background Color block. Since the block editor isn't initialized yet on domReady, the getCurrentPostType() was returning null so the block wasn't being unregistered when on a Volunteer Opportunity post.

Reply:
Nice work getting this resolved!